### DISCUSSION TITLE
Huawei With 3,3v relais

### DISCUSSION THREAD

**Discussion by rudolfp1979** [tbnobody/OpenDTU#2329] [General]
Weis jemand wann das Relais abschaltet bzw ob es das überhaupt macht und wenn ja unter welchend Bedingungen ich meine hier das power Signal. Es zieht korrekt an beim einschalten und beim restart aber ein ausschalten konnte ich bis jetzt nicht erkennen?

**Comment by ghost** [comment]
Hi,
wenn es ein bistabiles Relais ist, kann es je nach Typ Jahre dauern, bis es freiwillig ausschaltet, I.d.R. behalten die ihren letzten Schaltungszustand bei.
Eine genauere Produktbezeichnung und/ oder Fotos wären hilfreich...

**Reply by rudolfp1979** [reply]
![IMG_0157](https://github.com/user-attachments/assets/23dd6b0d-3df2-4791-879b-c1cf848f4ea0)
Hier ein foto

**Comment by ghost** [comment]
Ich denke, du siehst das wie ich:
Das Relais benötigt eine dauerhafte Betriebsspannung von 3,3 V an VCC und GND.
Angesteuert wird es über ein HIGH-Signal am IN-Eingang,
Sobald das HIGH-Signal weg ist, sollte das Relais abschalten.
Ohne Betriebsspannung sollte es auch nicht gehen, bzw. das Relais sollte ebenfalls abschalten, wenn die Betriebsspannung fehlt.
Hattes du etwas an dem Wechsler-Anschluss angeschlossen? Eine LED würde ja reichen. Möglicherweise ist das "Klicken" beim Ausschalten nur nicht hörbar.
Die LED wird nur das Signal am IN-Eingang anzeigen.
Ich habe kein ähnliches Relais hier, kann das nicht testen.
Hast du eine Ahnung, wofür die 2 Jumper gedacht sind?

Es ist ja ein Schaltverstärker auf der Platine: Optokoppler und Transistor, die könnten etwas abbekommen haben.
Was passiert, wenn du zum Beispiel noch einen PULL-Down Wiederstand von etwa 1 KOhm (3,3 mA zusätzliche Last am Steuerausgang deines Mikrocontrollers) parallel zu IN und GND anschließt, du kannst das auch erstmal mit 10 KOhm versuchen (0,3 mA).
Du kannst das Relais auch ohne µPC testen, einfach mal 3,3V und 0V abwechselnd auf den IN-Eingang legen.

**Reply by rudolfp1979** [reply]
Ich hatte das Huawei dran und der Lüfter läuft konstant auch auf Einstellung keine Leistung des Netzteils ich dachte es müsste dann ausschalten. Aber es schaltet sich aktuell nur aus wenn ich den Saft von der opendtu trenne

**Reply by ghost** [reply]
Huawei? Das ist ein Netzteil, Batterielader?
Dazu kann ich nichts beitragen.



> Aber es schaltet sich aktuell nur aus wenn ich den Saft von der opendtu trenne

Wenn du den IN-Pin abklemmst passiert nichts? Dann könnte die Relais-Platine defekt sein.

**Reply by rudolfp1979** [reply]
Doch dann schaltet es auch aus. Meine Frage ist eher wann und ob das steuersignal überhaupt aus bzw auf low gehen müsste.

**Reply by Solarteur** [reply]
Hast du diesen Wiki-Eintrag durchgelesen:
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Huawei-AC-PSU

Darin ist die Verwendung dieses Relais genau beschrieben.

Nebenbei: Deine Frage wäre im OpenDTU-OnBattery-Forum besser aufgehoben.

**Reply by rudolfp1979** [reply]
Genau danach bin ich vorgegangen es funktioniert auch soweit alles bis zu dem Punkt weshalb ich hier aufschlage. Schaltet die dtu den Power auf high oder auf low wenn das huawei angehen soll das kommt in dem Beitrag nicht wirklich raus. Aktuell hab ich es auf normale Open und gehe davon sus das Signal kommt und das Relais zieht an sollte die Software das Signal aber abschalten müsste ich ja auf normally closef gehen.

**Reply by rudolfp1979** [reply]
so von no auf nc damit tut es was es soll.
